src/client/src/containers/Footer/index.tsx

```diff
@@ -202,30 +202,28 @@ class Footer extends React.Component<Props> {
               >
                 <FormattedMessage id="footer.back" defaultMessage="Back" />
               </Link>
-              <Link
-                tabIndex={
-                  !isValidNameAndProjectPath || this.isReviewAndGenerate()
-                    ? -1
-                    : 0
-                }
-                className={classnames(styles.button, {
-                  [buttonStyles.buttonDark]:
-                    this.isReviewAndGenerate() || !isValidNameAndProjectPath,
-                  [buttonStyles.buttonHighlightedBorder]: !this.isReviewAndGenerate(),
-                  [styles.disabledOverlay]:
-                    !isValidNameAndProjectPath || this.isReviewAndGenerate()
```

Comment:
Is `isReviewAndGenerate` function still used somewhere? If not, it should be removed

Reply:
Yes, it is still being used in the Review and Generate button.